client/src/components/Container.jsx

```diff
@@ -111,11 +112,18 @@ class Container extends React.Component {
   visitItem(itemId) {
     window.location.pathname = `/items/${itemId}`;
   }
+
+  reviewImageClick(id) {
+    this.setState({
+      modalReview: _.findIndex(this.state.reviewImages, {review_id: parseInt(id)}),
+      showModal: true
+    });
+  }
   
   render() {
     return (
       <div style={containerStyles.container}>
-        <div style={{display: 'flex'}}>
+        <div style={{display: 'flex', marginBottom: '-10px'}}>
```

Comment:
How come you didn't use the imported css for this div, like the other divs

Reply:
time :) can definitely refactor